tabular/src/autogluon/tabular/models/lgb/lgb_model.py

```diff
@@ -404,7 +509,13 @@ def _features(self):
         return self._features_internal_list
 
     def _ag_params(self) -> set:
-        return {"early_stop"}
+        return {"early_stop", "generate_curves", "curve_metrics", "use_error_for_curve_metrics"}
+
+    @classmethod
+    def _class_tags(cls):
+        tags = super(LGBModel, cls)._class_tags().copy()
```

Comment:
No need to do this. This logic is already present. Instead, call `_get_class_tags` to get the tags.

```
    @classmethod
    def _class_tags(cls):
        return {"supports_learning_curves": True}

tags = LGBModel._get_class_tags()  # Will contain `{"supports_learning_curves": True}` along with any parent tags.
```